R/get_child_mortality.R

```diff
@@ -0,0 +1,161 @@
+#' Get child mortality rates
+#' 
+#' @description
+#' This function computes and returns the child mortality rates developed by the 
+#' UN Inter-Agency Group for Child Mortality Estimation, which includes UNICEF, 
+#' WHO, the World Bank, and the UN DESA Population Division. This data is retrieved
+#' from the World Bank website, and terms of use are found here: https://www.worldbank.org/en/about/legal/terms-of-use-for-datasets.
+#' 
+#' @details 
+#' Neonatal mortality: the probability of dying between the first 28 days of life, expressed per 1000 live births.
+#' Infant mortality: the probability of dying between birth and exact age 5, expressed per 1000 live births.
+#' Under 5 mortality rate: the probability of dying between birth and exact age 5, expressed per 1000 live births.
+#' 
+#' Future additions:
+#' Memory use improvements. Check of years and ages inputs. Uses at least an extra 2.6 MB. 
+#' Spell checks. Url check - make sure its still there.
+#' 
+#' @param country all, iso3c code, or country name in English. If not available or 
+#' misspelled, will throw error. To do: add a list of all countries/regions that we 
+#' have data for.
+#' @param gender both (only total), female, male, all (female, male, and total).
+#' Data by gender is only available for infant mortality and under 5 mortality.
+#' @param years.start Mortality data is available between 1960-2019. This specifies
+#' the first year for which values are returned, default is 1960.
+#' @param years.end Specified the last year for which values are returned. By default
+#' returns up to 2019.
+#' @param indicator Indicators for the function to return. Either returns all, 
+#' infant mortality ("infant"), neonatal mortality("neonatal"), or under 5 
+#' mortality ("under 5"). 
+#' 
+#' @keywords UN, demography, download, population, child, deathrates, infant, mortality, neonatal, under 5
+#' @return List of mortality rates.
+#' @export
+#' 
+#' @import countrycode
+#' @import tidyr
+#' @import data.table
+#' @import stringr
+#' @import tidyverse
+#' @importFrom dplyr bind_rows
+#' 
+#' @examples
+#' get_child_mortality(country="Norway")
+get_child_mortality <- function(country="all", 
```

Comment:
This seems a bit pedantic (sorry!) but good to follow a consistent style guide (e.g. spaces arounf <- and =).
The tidyverse style guide is a good reference here: https://style.tidyverse.org/index.html